lib/constants/category.ts

```diff
@@ -24,6 +24,7 @@ const Categories = [
   'rust',
   'swift',
   'terraform',
+  'unity3d',
```

Comment:
Unity has [a different package manager (Unity Package Manager)](https://docs.unity3d.com/Manual/upm-concepts.html) for its sub-dependencies and while I've read #19300, I haven't got it working using the proposed NPM workaround: There's a lock file different from normal NPM lock files that would have to be generated and currently isn't. This might be something worth adding under the `unity3d` umbrella.

Let me know if you find any existing ones more appropriate.

Reply:
Makes sense to add the category then, if support for this manager is added too. 

Please remove it for now.